### ISSUE TITLE
1Password autofill does not work on login

### ISSUE THREAD

**Issue by itsaphel** [mysociety/alaveteli#6376]
### Steps to reproduce
1. Go to https://www.whatdotheyknow.com/profile/sign_in
2. Click 1Password button and try to autofill credentials
3. It autofills half the info into the registration form (to the left) and other half into the login form (to the right)

May be a problem with other password managers too. Possibly relevant: https://support.1password.com/compatible-website-design/

Only tested on WDTK, not on a raw install of Alaveteli.

**Comment by mdeuk** [comment]
> 
> ### Steps to reproduce
> 
>     1. Go to https://www.whatdotheyknow.com/profile/sign_in
> 
>     2. Click 1Password button and try to autofill credentials
> 
>     3. It autofills half the info into the registration form (to the left) and other half into the login form (to the right)

I can't _consistently_ reproduce this using 1Password v7.7.810 (with Firefox addon version 4.7.5.90) - in this instance, on a Windows 10 install.

What fields has 1Password captured when you created your saved item?

**Comment by itsaphel** [comment]
Attaching a pic of how it looks for me once I fill out the form. 1Password v7.8.6 (extension v4.7.5.90) on macOS.

It does fill out the email, just in the wrong box for some reason.

![filled form](https://user-images.githubusercontent.com/2878052/125839287-a091618e-6ee9-4b31-aa40-3ec123a7f343.jpg)

**Comment by gbp** [comment]
I've been successfully using 1Password v6 on macOS with no issues. Both `user_signin[email]` and `user_signin[password]` are non-empty values captured from the Firefox extension.

**Comment by garethrees** [comment]
I think something is definitely up here. I've always had weird autofill behaviour (LastPass / Chrome / macOS) on _some_ Alavetelis.

![password-autofill](https://user-images.githubusercontent.com/282788/126479730-8196dbc2-3e5d-4ced-a337-fac437c2599c.jpg)

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
